data/core/logview.lua

```diff
@@ -123,11 +123,26 @@ end
 
 
 function LogView:update()
+  local user_scroll
```

Comment:
Bit of leftover.

Reply:
oh, had that removed from dev version, somehow slipped at staging setup, thanks.